### PR TITLE
ticket-741

### PR DESCRIPTION
**Pull request by butlerpd**
Fixes S * P being returned and made plottable - NOTE: this requires also branch ticket-741 in sasmodels

### CODE REVIEW

**Comment by butlerpd** [comment]
NOTE1: master was merged into this branch immediately prior to making this pull request
NOTE2: Paul Kienzle is not being listed as a reviewer as he essentially wrote most of this and agreed pn the final.

File: src/sas/sasgui/perspectives/fitting/model_thread.py [hunk 1]
```diff
@@ -197,25 +198,25 @@ def compute(self):
         else:
             output[index] = self.model.evalDistribution(self.data.x[index])
 
+        x=self.data.x[index]
```
Comment (andyfaff) [inline src/sas/sasgui/perspectives/fitting/model_thread.py]: space either side of the equal sign

Comment (andyfaff) [inline src/sas/sasgui/perspectives/fitting/model_thread.py]: and on next line

**Comment by RichardHeenan** [comment]
Also, if I append P(Q) and S(Q) to the fit plot, as expected they don't auto update on chaing parameters (yet), however if I hit "append to plot" again, I don't get new versions of the curves, but the ones that are there already then update.  Somewhere a user will get confused ... so the sooner we can get auto-updating, the better, but then that may change behaviour elsewhere, sop may need to be able to lock curves on a plot so they don't update ... oh this could get messy.

**Comment by RichardHeenan** [comment]
Hi Paul

Who is the best person to be looking into this one?

https://github.com/SasView/sasview/pull/89

Richard

**Comment by wpotrzebowski** [comment]
ready for testing on Linux

**Comment by wpotrzebowski** [comment]
ready for testing on Linux

**Comment by wpotrzebowski** [comment]
ready for testing on Linux

**Comment by wpotrzebowski** [comment]
ready for testing on Linux

**Comment by wpotrzebowski** [comment]
ready for testing on Linux

**Comment by butlerpd** [comment]
Responding to richard -- I am now part way to getting at least one development system up -- but to verifyt that when you pulled the ticket-741 branch are you sure you ALSO changed to the ticket-741 branch in sasmodels.  They work in tandem since we had to make changes to both SasView and sasmodels to make this work.  If you took ticket-741 of SasView only I am not surprised you got strange results.  

Can you verify?

**Comment by RichardHeenan** [comment]
Sorry Paul, I'm on vacation this week....
Richard

Sent from my Windows 10 phone

**Comment by RichardHeenan** [comment]
Hi Paul B
Thought I would try a quick look at this before I have to go home, did:
Git checkout FixingTicket741   for sasview
Git checkout ticket-741 for sasmodels
But when I build here the fitting perspective does not appear, not obvious errors building, but fails to find it when I run.
Perhaps not surprisingly, github says there may be conflicts, but are these the correct branches????

Richard

17:34:10 - ERROR : sas.sasview.sasview: 104: SasView: could not find Fitting plug-in module
17:34:10 - ERROR : sas.sasview.sasview: 105: Traceback (most recent call last):
  File "C:\sasview4\sasview\sasview\sasview.py", line 100, in __init__
    import sas.sasgui.perspectives.fitting as module
  File "C:\sasview4\sasview\src\sas\sasgui\perspectives\fitting\__init__.py", line 3, in <module>
    from fitting import *
  File "C:\sasview4\sasview\src\sas\sasgui\perspectives\fitting\fitting.py", line 45, in <module>
    from sas.sasgui.perspectives.calculator.model_editor import TextDialog
  File "C:\sasview4\sasview\src\sas\sasgui\perspectives\calculator\__init__.py", line 4, in <module>
    from calculator import *
  File "C:\sasview4\sasview\src\sas\sasgui\perspectives\calculator\calculator.py", line 25, in <module>
    from sas.sasgui.perspectives.calculator.gen_scatter_panel import SasGenWindow
  File "C:\sasview4\sasview\src\sas\sasgui\perspectives\calculator\gen_scatter_panel.py", line 32, in <module>
    from sas.sascalc.calculator import sas_gen
  File "C:\sasview4\sasview\src\sas\sascalc\calculator\sas_gen.py", line 7, in <module>
    import sas.sascalc.calculator.core.sld2i as mod
ImportError: DLL load failed: The specified procedure could not be found.

**Pull request closed without merging**